lib/plugins/aws/package/lib/validateCompiledTemplate.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const CF = require('aws-sdk/clients/cloudformation');
```

Comment:
@HyperBrain thanks for jumping in!

@hassankhan Yes, using the request object is the way to go (the main purpose of exposing it was that other plugin authors can use and rely on the same `sdk` the core uses).